File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/LocationsLocation.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class LocationsLocation {
    @JsonProperty("ForHarvests")
    public Boolean forHarvests;
    @JsonProperty("ForPackages")
    public Boolean forPackages;
    @JsonProperty("ForPlantBatches")
    public Boolean forPlantBatches;
    @JsonProperty("ForPlants")
    public Boolean forPlants;
    @JsonProperty("Id")
    public Integer id;
    @JsonProperty("LocationTypeId")
    public Integer locationTypeId;
    @JsonProperty("LocationTypeName")
    public String locationTypeName;
    @JsonProperty("Name")
    public String name;
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/LocationsLocation.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class LocationsLocation(
    @JsonProperty("ForHarvests")
    val forHarvests: Boolean? = null,
    @JsonProperty("ForPackages")
    val forPackages: Boolean? = null,
    @JsonProperty("ForPlantBatches")
    val forPlantBatches: Boolean? = null,
    @JsonProperty("ForPlants")
    val forPlants: Boolean? = null,
    @JsonProperty("Id")
    val id: Int? = null,
    @JsonProperty("LocationTypeId")
    val locationTypeId: Int? = null,
    @JsonProperty("LocationTypeName")
    val locationTypeName: String? = null,
    @JsonProperty("Name")
    val name: String? = null
)
